build.bat

```diff
@@ -1,5 +1,4 @@
 @ECHO OFF
 set DOTNET_CLI_TELEMETRY_OPTOUT=1
 dotnet restore .\RabbitMQDotNetClient.sln
-dotnet run -p .\projects\Apigen\Apigen.csproj --apiName:AMQP_0_9_1 .\projects\specs\amqp0-9-1.stripped.xml .\gensrc\autogenerated-api-0-9-1.cs
```

Comment:
Build is still failing. Am I missing something or does this not have the desired effect?

Reply:
I had to fix some scripts - https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/924#issuecomment-676561180